packages/frontend/src/translations/en.global.json

```diff
@@ -320,16 +320,22 @@
         "ledgerConnected": "Ledger Connected",
         "modal": {
             "connect": {
+                "cancelButton": "Cancel",
+                "connectButton": "Connect",
                 "header": "Connect to your Ledger device.",
                 "one": "Make sure your Ledger is connected securely, and that the NEAR app is open on your device."
             },
+            "connection-error": {
+                "cancelButton": "Cancel",
+                "connectButton": "Retry",
+                "header": "Connection unsuccessful.",
+                "one": "Make sure your ledger is connected securely, and that the NEAR app is open on your device."
```

Comment:
Can we please rename these `"one"` keys? `one` is not a descriptive name and although we use it in other places I think it's a habit we should break in order to produce code that is easier to read and debug.

Something like `connectionPrompt` would work well here I think, but really anything that describes what this text indicates is viable.

Reply:
Done.